src/test/it/runJobs.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/sh
+buildJob(){
+  local job=${1:-?}
+  echo $(curl -s -o /dev/null -w '%{http_code}' http://127.0.0.1:8080/job/${job}/build?delay=0sec)
+}
+
+waitFinishJob(){
+  local job=${1:-?}
+  RUNNING="<building>true</building>"
+  while [ "${RUNNING}" = "<building>true</building>" ] 
+  do
+    sleep 10
+    RUNNING="$(curl -sS http://127.0.0.1:8080/job/${job}/1/api/xml?xpath=/workflowRun/building)"
+    echo -n "."
+  done                     
+}
+
+getResultJob(){
+  local job=${1:-?}
+  echo "$(curl -sS http://127.0.0.1:8080/job/${job}/1/api/xml?xpath=/workflowRun/result)"
+}
+
+getDurationJob(){
+  local job=${1:-?}
+  echo "$(curl -sS http://127.0.0.1:8080/job/${job}/1/api/xml?xpath=/workflowRun/duration)"
+}
+
+deleteJob(){
+  local job=${1:-?}
+  echo "$(curl -s -o /dev/null -w '%{http_code}' http://127.0.0.1:8080/job/${job}/doDelete)"
```

Comment:
You must POST to this. In fact you _are_ getting a 405 response—but the test passes anyway. So this `curl` usage is not actually verifying anything AFAICT.

Reply:
the only not tested results are for doDelete operations, all the rest check the return test or code.